src/unity/schemas/Account.yml

```diff
@@ -24,11 +13,4 @@ properties:
   balance:
     type: number
     description: remaining balance on the account, nil if none
-  users:
-    type: array
-    items:
-      $ref: './User.yml'
-  billingContact:
-    $ref: './BillingContact.yml'
-    description: billing contact for the account
-required: [id, balance, billingContact, marketplace, users, type]
+required: [id, balance, type]
```

Comment:
different places where accounts are used have different required fields and nested resource. those are defined in the location of use